addon-test-support/invocation-strategies.ts

```diff
@@ -0,0 +1,21 @@
+export interface InvocationStrategy {
+  (): boolean;
+}
+
+export const invokeAll: InvocationStrategy = function invokeAll(): boolean {
+  return true;
+};
+
+export function invokeEveryN(interval: number): InvocationStrategy {
```

Comment:
Maybe `invokePercentange`?

Reply:
I felt like every n tests was easier to think about. Why do you prefer percentage?

Reply:
In fact, I think `n tests` is more precise, as it's actually invoking every n test. No fake news :trollface:!

Reply:
We've moved this to an example instead.